src/mlpack/tests/image_load_test.cpp

```diff
@@ -12,6 +12,9 @@
 
 #include <mlpack/core.hpp>
 #include <boost/test/unit_test.hpp>
+#include <memory>
```

Comment:
Not sure I see the need for the include, but maybe I missed something?

Reply:
done